metadata/pkg/drivers/aws/aws.go

```diff
@@ -134,7 +167,17 @@ func GetBucketMeta(buckIdx int, bucket *s3.Bucket, sess *session.Session, bucket
 	} else if !strings.Contains(err.Error(), "NoSuchTagSet") {
 		log.Errorf("unable to get bucket tags. failed with error: %v", err)
 	}
-	bucketArray[buckIdx] = buck
+
+	acl, err := svc.GetBucketAcl(&s3.GetBucketAclInput{Bucket: bucket.Name})
+	if err != nil {
+		log.Errorf("unable to get bucket Acl. failed with error: %v", err)
+	} else {
+		access := make([]*model.Access, len(acl.Grants))
```

Comment:
make may not required

Reply:
make is being used for map initialisation.